content/ngf/installation/installing-ngf/upgrading-ngf.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Upgrade NGINX Gateway Fabric to version 2.0.0
+weight: 400
+toc: true
+type: how-to
+product: NGF
+docs: DOCS-0000
+---
+
+This guide provides step-by-step instructions for upgrading NGINX Gateway Fabric from version 1.6.x to 2.0.x, highlighting key architectural changes, expected downtime, and important considerations for custom resource definitions.
+
+---
+
+### Upgrade from v1.6.x to v2.0.x
+
+To upgrade NGINX Gateway Fabric from version 1.6.2 to the latest version 2.0.0, you must uninstall the existing CRDs and perform a fresh installation with the new CRDs. This upgrade involves uninstalling the existing NGINX Gateway Fabric deployment making it a breaking change and will lead to application downtime during the process.
```

Comment:
```suggestion
To upgrade NGINX Gateway Fabric from version 1.6.2 to the latest version 2.0.0, you must uninstall the existing NGINX Gateway Fabric CRDs and deployment, and perform a fresh installation. This will cause brief downtime during the upgrade process.
```